packages/admin/src/middleware/Auth.middleware.ts

```diff
@@ -20,8 +20,14 @@ async function requestExcluded(req: ConduitRequest, conduit: ConduitCommons) {
   if (excludedRoutes.includes(req.path)) return true;
   // GraphQL
   if (excludedGqlMutations.includes(req.body.operationName)) return true;
-  const queryName = req.body.query?.split('{\n')[1].split('\n}')[0].trim();
-  if (excludedGqlQueries.includes(queryName)) return true;
+  if (
+    req.body.query &&
+    typeof req.body.query === 'string' &&
+    req.body.query.startsWith('{\n')
```

Comment:
These checks require the existence of a special character "\n" to work. This doesn't seem stable at all, we don't care about general content just the existence of a specific query. This check needs to be changed to reflect that in a more concrete manner